newsletter/11.04.2020_newsletter.md

```diff
@@ -0,0 +1,45 @@
+---
+title: November 04, 2020 Newsletter
```

Comment:
This can probably just say "November 4"

Reply:
I thought this complies with the current format?

Reply:
The title is the heading at the top of each page. Scrolling through the last few newsletters, it looks like we don't include a zero in the date

Reply:
I see what you mean now. Title changed to `November 4`.